### DISCUSSION TITLE
Support github.base_ref for pull_request_review

### DISCUSSION THREAD

**Discussion by nitzanashi** [community/community#26006] [Actions]
<p>currently <code>github.base_ref</code> can be used only on <code>pull_request</code> workflow.<br>
It will be great if we can also access it on <code>pull_request_review</code> events</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/nitzanashi">@nitzanashi</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>The two events have different Github Context. For ‘pull_request_review’ event, base ref can be got from <code>${{ github.event.pull_request.base.ref }}</code>.</p>
<p>You can add below code in workflow to export Github Context details:</p>
<pre><code class="lang-auto">      - name: Dump GitHub context
        env:
          GITHUB_CONTEXT: ${{ toJson(github) }}
        run: echo "$GITHUB_CONTEXT"
</code></pre>
<p>Thanks</p>
